samples/peripheral/802154_phy_test/boards/nrf21540dk_nrf52840.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SHELL_PROMPT_UART=">"
```

Comment:
Missing license. I know this file contains one trivial line but I'm assuming we always have to publish licensed files.

Reply:
amended